phoenix-core/src/it/java/org/apache/phoenix/end2end/End2EndTestDriver.java

```diff
@@ -117,7 +117,7 @@ protected void addOptions() {
 
     @Override
     protected void processOptions(CommandLine cmd) {
-      String testFilterString = cmd.getOptionValue(SHORT_REGEX_ARG, null);
+      String testFilterString = cmd.getOptionValue(SHORT_REGEX_ARG, (String)null);
```

Comment:
we could remove the 2nd argument altogether, as the default value is anyways null. https://commons.apache.org/proper/commons-cli/apidocs/org/apache/commons/cli/CommandLine.html#getOptionValue(char)

Reply:
Yes, that's more elegant.

Reply:
Removed explcit default.